Review for #84

unode opened the pull request:
Minor corrections during issue #84

unode committed:
Expand 's contractions for "has"

tobyhodges commented:
I've marked this as a draft so it won't get merged before you're ready. Mark it "ready for review" once you're done 🙂

unode commented:
Ah!, my overuse of GitLab creeping up in this PR title :smile:
Thanks Toby

unode committed:
Add link to git

unode committed:
Request readers to login in case they haven't

unode committed:
Avoid the word 'repo' before introduction

unode committed:
Introduce the word 'repo'

tobyhodges commented on _includes/links.md:
```diff
@@ -16,6 +16,7 @@
 [email]: mailto:team@carpentries.org
 [filters]: https://jekyllrb.com/docs/liquid/filters/
 [ghp-dependencies]: https://pages.github.com/versions/
+[git]: https://git-scm.com/
```
I guess we should link to the git homepage, but please also find a way to add a link to https://swcarpentry.github.io/git-novice/ in this section

unode replied:
Added.

unode committed:
Add reference to SWC git lesson from introduction

unode committed:
Link first mention of 'indentation' to wikipedia

unode committed:
Add single backtick and that they should surround the text

unode committed:
Mention the green vertical bars in GitHub's interface

unode committed:
Add missing formatting

unode committed:
Correct solution to show the same numbering as the screenshot

It's a good idea to show that the numbers in markdown don't necessarily reflect
the numbers in the rendered output but without explaining this before finding
a 5. in the solution is confusing and wouldn't be 'guessable'.

unode committed:
Provide some context to the linking exercise

unode committed:
Mention topics/tags in making pages more findable

unode committed:
Format code symbols with backticks

unode committed:
example-lesson -> lesson-example

unode committed:
Add missing words and remove unnecessary characters

unode committed:
Fix incorrect image path - shouldn't start with /

unode committed:
Correct also the hyperlinked image

unode commented on _episodes/includes.md:
````diff
@@ -149,7 +149,7 @@ at the top of your page.
 > >
 > > ```
-> > <a href="https://YOUR_USERNAME.github.io/YOUR_REPO_NAME/"><img src="/images/site_banner.png" alt="Group Website with Jekyll"></a>
+> > <a href="https://YOUR_USERNAME.github.io/YOUR_REPO_NAME/"><img src="images/site_banner.png" alt="Group Website with Jekyll"></a>
````
I can see this break if the include done later is done from another folder. Either way, neither prefixing with / nor leaving it out is going to work for all cases.

unode replied:
Wondering if it makes sense to turn this into an exercise in "Page layouts" as part of "Expanding the layout".

unode committed:
team.md should be about.md

unode commented:
Sorry for the delay in wrapping this up.

Noticed after that @annefou did the review in issues, some of the things in this pull request have been pointed out in #113, #114, #115 and #116 as well.

unode committed:
Merge branch 'gh-pages' into review-renato-84

unode commented:
Fixed conflicts. Ready for review/merge

tobyhodges commented on _episodes/github-pages.md:
```diff
@@ -168,8 +169,8 @@ We are now ready to start adding more content to our website. Let's do some exer
-> >     Note how we used various Markdown syntax: quoted text ('>'), italic font ('*') and external links
-> >     (square '[]' and round brackets '()' for mailto and regular Web URLs).
+> >     Note how we used various Markdown syntax: quoted text (`>`), italic font (`*`) and external links
+> >     (square `[]` and round brackets `()` for mailto and regular Web URLs).
```
```suggestion
> >     (a combination of square `[]` and round brackets `()` containing the link text and mailto or regular Web URLs respectively).
```

I think this is clearer - the original wording could potentially be confusing as a learner might read it as meaning "use square brackets for mailto and regular brackets for Web URLs."

tobyhodges reviewed (approved):
One suggested change but otherwise this looks great. Thanks @unode

unode committed:
Update _episodes/github-pages.md

Co-authored-by: Toby Hodges <tbyhdgs@gmail.com>
